src/semgrep_agent/semgrep_app.py

```diff
@@ -153,14 +153,19 @@ def report_failure(self, error: SemgrepError) -> int:
     def report_results(self, results: Results) -> None:
         debug_echo(f"=== reporting results to semgrep app at {self.url}")
 
+        fields_to_omit = constants.PRIVACY_SENSITIVE_FIELDS.copy()
+
+        if "pr-comment-autofix" in os.getenv("SEMGREP_AGENT_OPT_IN_FEATURES", ""):
```

Comment:
I am having trouble imaging a world where a user does not want to opt in here - should we just make it leave autofix recommendations any time PR comments are enabled?

Reply:
This was because we promised not to collect customer source code, and I just remembered I still needed to clear this change with people.

Reply:
Oh, I thought we were ok as long as we didn't store their code anywhere - is this different than what we output to them in the terminal?

Reply:
I worry that anything from this JSON blob is saved as findings data. We'll need to strip this in the backend to prevent it from being stored, and even after that we should clear this with someone with a better understanding of privacy policies.